src/pages/home/report/ReportActionItem.js

```diff
@@ -254,6 +264,14 @@ class ReportActionItem extends Component {
                         />
                     </View>
                 )}
+                {shouldDisplayThreadReplies && (
+                    <ReportActionItemThread
```

Comment:
👋This has caused a small visual issue in https://github.com/Expensify/App/issues/19398.
`Reply` text wasn't aligned with composer when editing, ReportActionItemThread should have the same styles applied as ReportActionItemReactions above: `this.props.draftMessage ? styles.chatItemReactionsDraftRight : {}`